wagtailio/static/sass/base/_base.scss

```
@@ -40,7 +40,7 @@ textarea {
     font-family: inherit;
 }
 
-fieldset {
+div, fieldset {
```

This is puzzling to me – `div` elements have no border or padding set by default.

```suggestion
fieldset {
```